### DISCUSSION TITLE
obs-websocket on linux

### DISCUSSION THREAD

**Discussion by alpham8** [Homebrew/discussions#2808] [Casks]
Hi,

if I try to run this command on Linux with Linuxbrew, it fails:
```
brew install --cask obs-websocket
```
I'd like to install it on Linux (openSUSE).

Any help to this would be appreciated.

**Comment by gromgit** [comment]
The error message is quite clear about why it fails:
```
$ brew install --cask obs-websocket
Error: Installing casks is supported only on macOS
```
> I'd like to install it on Linux (openSUSE).

You want to visit the project home page (`brew home obs-websocket`) and see what options there are for Linux installation outside of Homebrew.

**Reply by alpham8** [reply]
I have no other options, only compile it myself, because I'm on openSUSE and they do not build an rpm package, only deb.

**Reply by carlocab** [reply]
We'd be happy to package it if you contribute a formula. It looks like `obs-websocket` depends on `libobs`, though, so we'd need a formula that provides that first.

**Reply by alpham8** [reply]
@carlocab Well, that's why I went to Homebrew, because I need otherwise to compile it on my own and I'm missing one package from my dsitributor. It's qt5-image-formats-plugins, maybe it's already in Homebrew, so a combination from that Fomula would be possible? But usually compiling is overwhelming me, that's why I'm glad that Homebrew exists

**Comment by alpham8** [comment]
Sorry, I should have posted it at the beginning. For me it is:
```
brew install --cask obs-websocket
==> Auto-updated Homebrew!
Updated 1 tap (homebrew/core).
==> Updated Formulae
Updated 2 formulae.
==> Deleted Formulae
gr-osmosdr

Error: Cask 'obs-websocket' is unavailable: No Cask with this name exists.
```

Are there any options to get this formula on Linux?

**Reply by gromgit** [reply]
Ah, sorry, I ran that command on macOS, for which the Cask repo would've been tapped, hence the different error message.

The fundamental issue hasn't changed, though: Casks aren't supported on Linux, period. (The error you saw could be improved, to be honest.)

I've already mentioned one solution in my previous comment. The only other alternative is for someone to write a formula (_not_ a cask) for `obs-websocket`, though I suspect there's a show-stopper that explains why this hasn't already been done.
